pkg/machine/qemu/machine.go

```diff
@@ -589,11 +588,7 @@ func (v *MachineVM) qemuPid() (int, error) {
 		logrus.Warnf("Reading QEMU pidfile: %v", err)
 		return -1, nil
 	}
-
-	if err := unix.Kill(pid, 0); err != nil {
-		if err == unix.ESRCH {
-			return -1, nil
-		}
+	if err := killIfFound(pid); err != nil {
```

Comment:
this is not equivalent, now it will not return -1,nil for ESRCH which I assume is incorrect.

Reply:
oops, thanks!